tests/containerTests.js

```diff
@@ -34,6 +34,15 @@ describe('Container', () =>  {
             expect(foo.bar).toBe(5);
         });
 
+        it('should register/resolve an object with a custom factory', () =>  {
+            let Foo = function(){};
```

Comment:
I tried it and it doesn't work. That way what I have is an object instance, not a constructor, so when I call `new Foo` it breaks.